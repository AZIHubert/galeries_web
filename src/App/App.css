@font-face {
    font-family: 'HelveticaLTstd';
    font-style: normal;
    font-weight: normal;
    src: url('../ressources/fonts/HelveticaLTStdRoman.ttf');
}
@font-face {
    font-family: 'HelveticaLTstd';
    font-style: normal;
    font-weight: bold;
    src: url('../ressources/fonts/HelveticaLTStdBold.ttf');
}
@font-face {
    font-family: 'HelveticaLTstd';
    font-style: italic;
    font-weight: normal;
    src: url('../ressources/fonts/HelveticaLTStdObl.ttf');
}
@font-face {
    font-family: 'HelveticaLTstd';
    font-style: normal;
    font-weight: lighter;
    src: url('../ressources/fonts/HelveticaLTStdLight.ttf');
}
