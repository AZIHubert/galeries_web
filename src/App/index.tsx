import * as React from 'react';
import { Reset } from 'styled-reset';

import Home from '#containers/Home';

import { UserProvider } from '#contexts/UserContext';

const App = () => (
  <UserProvider>
    <Reset />
    <Home />
  </UserProvider>
);

export default App;
