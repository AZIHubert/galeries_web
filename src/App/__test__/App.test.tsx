import * as React from 'react';
import { cleanup } from '@testing-library/react';
import renderer from 'react-test-renderer';

import App from '../index';

describe('App', () => {
  afterEach(cleanup);
  it('renders without crashing', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
