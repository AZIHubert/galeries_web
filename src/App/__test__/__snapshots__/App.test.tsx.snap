// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders without crashing 1`] = `
<div
  className="sc-cbDGPM hDGbnb"
>
  <div
    className="sc-jrAGrp fcftTm"
  >
    <img
      alt="Galeries logo"
      className="sc-kEjbxe fWOire"
      src={Object {}}
    />
  </div>
</div>
`;
